migration-guides/v6.0.0.md

```diff
@@ -77,3 +78,14 @@ Any `<RangeInput />` with a `hideLabel` prop should be updated to use `hideRange
 // After
 <RangeInput input={rangeInputProps} hideRangeValue={true} />
 ```
+#### 6. The tag on `<Spinner />` now uses the classname `spinner` in place of an id
```

Comment:
Can you merge in the latest from v6 into your branch? 6, 7, and 8 were recently added